src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableTestWarningSeverity.java

```diff
@@ -0,0 +1,17 @@
+/*
+UnusedLocalVariable
+severity = warning
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unusedlocalvariable;
+
+public class InputUnusedLocalVariableTestWarningSeverity {
+
+    void m() {
+     @Test.A Outer p1 = new @Test.A Outer();
+     @Test.A Outer.@Test.B Inner p2 = p1.new @Test.B Inner(); // ok until, issue #12980
```

Comment:
Place exact link to issue as mentioned previously, this guarantees that CI job picks up closed issue references. `// ok, until https://github.com/checkstyle/checkstyle/issues/12980`

Reply:
@nrmancuso DONE

There was 100 characters limit that why i did that. Now I've mentioned it on the next line as:
```java
 void m() {
     @Test.A Outer p1 = new @Test.A Outer();
     @Test.A Outer.@Test.B Inner p2 = p1.new @Test.B Inner();
     // ok above until https://github.com/checkstyle/checkstyle/issues/12980
    }
```